e2e-tests/oauth-flow/rfc021/run-test.sh

```diff
@@ -66,14 +66,14 @@ fi
 echo "------------------------------------"
 echo "Retrieving data..."
 echo "------------------------------------"
-#RESPONSE=$(docker compose exec nodeB curl --insecure --cert /etc/nginx/ssl/server.pem --key /etc/nginx/ssl/key.pem https://nodeA:443/ping -H "Authorization: bearer $(cat ./node-B/data/accesstoken.txt)" -v)
-#if echo $RESPONSE | grep -q "pong"; then
-#  echo "success!"
-#else
-#  echo "FAILED: Could not ping node-A" 1>&2
-#  echo $RESPONSE
-#  exitWithDockerLogs 1
-#fi
+RESPONSE=$(docker compose exec nodeB curl --http1.1 --insecure --cert /etc/nginx/ssl/server.pem --key /etc/nginx/ssl/key.pem https://nodeA:443/resource -H "Authorization: bearer $(cat ./node-B/data/accesstoken.txt)" -v)
```

Comment:
should we also have a not-OK test?

Reply:
I'd like to have e2e tests to make sure the integration isn't broken at some point. Various error conditions should be tested by unit-tests. At this point a no-OK test would largely test the nginx config.